CenterForOpenScience/ember-osf-web
# [ENG-197] Revamp overview topbar

## Purpose

Revamp registries overview topbar

## Summary of Changes

## Side Effects

## Feature Flags

## QA Notes
❗️This topbar should not show for withdrawn and archiving registrations.

The topbar has two part:
1) states - left side
- The trigger button shows the button text for the current state - visible to all
- When clicked, you should see text describing the current registration state:
  - icon:
    pending states: clock outline icon
    embargoed: lock icon
    public: eye
  - header: represents the actual registration state. See `button text` on the table linked below
  - body of the text: long description of the current registration state. See `long description` column on table linked below
  - footer: admin actions. Only the registration admins should see action buttons.
      Admin actions: We currently only support ending embargo early and  withdrawing the registration for embargoed and public registration respectively. See `admin actions` on the table linked below
- For admins, the description/actions dropdown should automatic open when the overview page fully loads for all registration states except `public` and `embargoed`. See `Open popover on load` column on the table linked below.

Please refer to this [table](https://www.notion.so/cos/720f6cfd6ed54a98951be19e1af66dc9?v=fc1d748b30034d2090636c85561de408) for some clarifications on the above.

2) fork / bookmark / share - right side
- Make sure all the button have description tooltips
- Bookmark button should be disabled when using a VOL or AVOL.
- Check functionality of buttons and make sure no errors.

## Ticket

[ENG-197](https://openscience.atlassian.net/browse/ENG-197)

# Reviewer Checklist
